src/locales/pt-br/translation.json

```diff
@@ -0,0 +1,181 @@
+{
+  "1 Day": "1 Dia",
+  "1 Month": "1 Mês",
+  "1 Week": "1 Semana",
+  "2 Weeks": "2 Semanas",
+  "3 Days": "3 Dias",
+  "3 Months": "3 Meses",
+  "6 Months": "6 Meses",
+  "Accept": "Aceitar",
+  "Acknowledge": "Confirmar",
+  "acknowledged": "confirmado(s)",
+  "Acknowledged": "Confirmado(s)",
+  "Acknowledgments": "Confirmações",
+  "Actions": "Ações",
+  "Add Note": "Adicionar Anotação",
+  "Add Note to incident(s) here": "Adicionar Anotação a incidente(s) aqui",
+  "Add Responders": "Adicionar Respondentes",
+  "Alert Custom Detail Column Definitions": "Coluna de Definições dos Detalhes Personalizados do Alerta",
+  "Alert Custom Detail Column Definitions Placeholder": "Marcador de Coluna de Definições dos Detalhes Personalizados do Alerta",
+  "and their alerts have been merged onto incident": "e os seus alertas foram combinados no incidente",
+  "Assignees": "Detentores",
+  "Auto Accept Incident Query": "Consulta de incidente de aceitação automática",
+  "Auto Refresh Interval (mins)": "Intervalo de atualização automática (min)",
+  "Available": "Disponível",
+  "because they have already been suppressed or resolved": "porque já foram eliminados ou resolvidos",
+  "Cancel": "Cancelar",
+  "characters remaining": "caracteres restantes",
+  "Class": "Classe",
+  "Clear Local Cache": "Limpar Cache Local",
+  "Column Selector": "Seletor de Coluna",
+  "Component": "Componente",
+  "Connect using PagerDuty OAuth to use this app": "Conectar usando PagerDuty OAuth para usar esta aplicação",
+  "Connected": "Conectado",
+  "Connecting": "Conectando",
+  "Continue": "Continuar",
+  "Created At": "Criado Em",
+  "critical": "urgente",
+  "Current query parameters match X incidents": "Os parâmetros de consulta atuais coincidem com {{totalIncidentsFromQuery}} incidentes",
+  "Current subdomain does not have the correct ability to use PagerDuty Live": "O subdomínio atual não tem a capacidade correta para usar PagerDuty Live",
+  "Custom": "Personalizado",
+  "Custom Incident Action": "Ação de Incidente Personalizada",
+  "Decline": "Recusar",
+  "Default Since Date Lookback": "Retrospetiva a partir de Data Predefinida",
```

Comment:
We should change to "Retrospectiva" instead of "Retrospetiva"

Reply:
```suggestion
  "Default Since Date Lookback": "Retrospectiva a partir de Data Predefinida",
```